aws-http-nio-client-netty/pom.xml

```diff
@@ -50,6 +50,8 @@
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-transport-native-epoll</artifactId>
+            <version>${netty.version}</version>
+            <classifier>linux-x86_64</classifier>
```

Comment:
I'm saying we don't even need to include it as a dependency, but we give customers guidance, hey if you want to benefit from Netty's native epoll implementation and you're deploying on linux - then include this runtime dependency.

Or if you're deploying to BSD and you want to use the kqueue implementation include this one.